.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Fixes #
+
+For data ingests:
+[ ] includes script used for ingest
+[ ] includes modified JSON files
```

Comment:
At one point we were doing JSON files as part of a second PR, do we want to still do that or keep it in the same PR? The advantage of it being together is that it's easier for the folks preparing the PR, the disadvantage is that there maybe a lot of files to review.

Reply:
The 2nd PR for the JSON files seems silly. I've just been getting the ingest script reviewed before generating the JSON files.